iphone/Maps/Bookmarks/BookmarksList/BookmarksListPresenter.swift

```diff
@@ -182,6 +188,10 @@ extension BookmarksListPresenter: IBookmarksListPresenter {
     view.setInfo(info)
   }
 
+  func viewDidAppear() {
+    reload()
```

Comment:
Why is it needed now?

Reply:
There was an old bug with track title update (current AppStore version has it too).

If the user had edited and saved the track name it was not updated in the UI: popover window with track details will have the same title as before. 
It was the problem with title only (with both titles - fixed and flexible), if user, for example, has edited the description - it was updated in UI.  
Commit b818a188a0c2c1fbd0530cc8ccbf4bf2c468c76d solves this problem.

`reload()` in `viewDidAppear` solves the other part of this old problem: if user edits the title, then closes popover, then taps bookmarks button - previously opened bookmark group will appear. And if edited bookmark was in this group - it will have the old name. `reload` fixes that.